interfaces/vrep/DQ_SerialVrepRobot.m

```diff
@@ -0,0 +1,240 @@
+% CLASS DQ_SerialVrepRobot - Concrete class to interface with serial robots
+% in VREP.
+%
+% Usage:
+%       1) Drag-and-drop a serial robot to a VREP scene. For instance, a
+%       "my_robot" robot.
+%       2) Run
+%           >> vi = DQ_VrepInterface();
+%           >> vi.connect('127.0.0.1',19997);
+%           >> vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot", vi);
+%           >> vi.start_simulation();
+%           >> vrep_robot.get_q_from_vrep();
```

Comment:
This is probably the old usage `get_q_from_vrep`, better to update

Reply:
Updated in [0fd50ac](https://github.com/dqrobotics/matlab/pull/108/commits/0fd50ac04e454399a7b2d588758ee59b7cb2e533).